scheduler/policy.c

```diff
@@ -116,7 +116,7 @@ cupsdCheckPolicy(cupsd_policy_t *p,	/* I - Policy */
   {
     cupsdLogMessage(CUPSD_LOG_CRIT, "cupsdCheckPolicy: p=%p, con=%p.", p, con);
 
-    return ((http_status_t)0);
+    return (HTTP_STATUS_OK);
```

Comment:
Returning OK here isn't correct. Either change to `HTTP_STATUS_NONE` or maybe `HTTP_STATUS_SERVER_ERROR` instead since failure means a program error or (below) configuration error.